templates/welcome.php

```diff
@@ -0,0 +1,104 @@
+				'People can follow you by using the username %1$s or the URL %2$s. Users who can not access this settings page will find their username on the %3$sEdit Profile%4$s page.',
+				'activitypub'
+			),
+			\sprintf(
+				'<code>%s</code>',
+				\esc_attr( \Activitypub\get_webfinger_resource( wp_get_current_user()->ID ) )
+			),
+			\sprintf(
+				'<code>%s</code>',
+				\esc_url_raw( \get_author_posts_url( wp_get_current_user()->ID ) )
+			),
+			\sprintf(
+				'<a href="%s">',
+				\esc_url_raw( \admin_url( 'profile.php#activitypub' ) )
+			),
+			'</a>'
+		);
+		?>
+	</p>
+	<p>
+		<?php
+		\printf(
+			// translators:
+			\esc_html__( 'If you have problems using this plugin, please check the %1$sSite Health%2$s to ensure that your site is compatible and/or use the "Help" tab (in the top right of the settings pages).', 'activitypub' ),
+			\sprintf(
+				'<a href="%s">',
+				\esc_url_raw( admin_url( '/wp-admin/site-health.php' ) )
+			),
+			'</a>'
+		);
+		?>
+	</p>
+
+	<hr />
+
+	<h3><?php \esc_html_e( 'Recommendet Plugins', 'activitypub' ); ?></h3>
+
+	<p><?php \esc_html_e( 'ActivityPub works as is and there is no need for you to install additional plugins, nevertheless there are some that extends the functionality of ActivityPub.', 'activitypub' ); ?></p>
+
+	<div class="activitypub-settings-accordion">
+		<h4 class="activitypub-settings-accordion-heading">
+			<button aria-expanded="true" class="activitypub-settings-accordion-trigger" aria-controls="activitypub-settings-accordion-block-friends-plugin" type="button">
+				<span class="title"><?php \esc_html_e( 'Friends Plugin', 'activitypub' ); ?></span>
+				<span class="icon"></span>
+			</button>
+		</h4>
+		<div id="activitypub-settings-accordion-block-friends-plugin" class="activitypub-settings-accordion-panel">
+			<p><?php \esc_html_e( 'To follow people on Mastodon or similar platforms using your own WordPress, you can use the Friends Plugin for WordPress which uses this plugin to receive posts and display them on your own WordPress, thus making your own WordPress a Fediverse instance of its own.', 'activitypub' ); ?></p>
+			<p><a href="https://wordpress.org/plugins/friends" class="button"><?php \esc_html_e( 'Install Friends Plugin for WordPress', 'activitypub' ); ?></a></p>
+		</div>
+		<h4 class="activitypub-settings-accordion-heading">
+			<button aria-expanded="false" class="activitypub-settings-accordion-trigger" aria-controls="activitypub-settings-accordion-block-activitypub-hum-plugin" type="button">
+				<span class="title"><?php \esc_html_e( 'Hum Plugin', 'activitypub' ); ?></span>
+				<span class="icon"></span>
+			</button>
+		</h4>
+		<div id="activitypub-settings-accordion-block-activitypub-hum-plugin" class="activitypub-settings-accordion-panel" hidden="hidden">
+			<p><?php \esc_html_e( 'Hum is a personal URL shortener for WordPress, designed to provide short URLs to your personal content, both hosted on WordPress and elsewhere.', 'activitypub' ); ?></p>
```

Comment:
```suggestion
			<p><?php \esc_html_e( 'Hum is a self-hosted URL shortener for WordPress, designed to provide short URLs to your personal content, both hosted on WordPress and elsewhere.', 'activitypub' ); ?></p>
```

Clarify that it is self-hosted. Maybe add that it means that if you won't be affected if a URL shortener goes down?